service/internal/github/github.go

```diff
@@ -0,0 +1,116 @@
+package github
+
+import (
+	"context"
+
+	"github.com/giantswarm/config-controller/service/github/internal/graphql"
```

Comment:
```suggestion
	"github.com/giantswarm/config-controller/service/internal/github/internal/graphql"
```

Reply:
Also, for my knowledge. Why are there so many nested `internal`  folders? And why don't we put github/graphql package into `pkg`? As it is going to be used in cli.

Reply:
Thanks. Maybe I went a little bit wild. I'll move to `/pkg` then.